docs/docs/non-cloud-deployment.md

```diff
@@ -0,0 +1,115 @@
+
+# Non-cloud deployment
+
+Conclave recommends users to [deploy applications in Azure VMs](machine-setup.md) as it's the easiest method. 
+
+However, you can also deploy your enclaves on-premise.
+
+If your hardware supports EPID attestation instead of DCAP, you need to get access to the
+[Intel Attestation Service (IAS)](ias.md).
+
+To configure the host:
+
+1. Install the SGX kernel driver (not required for Linux kernel versions later than 5.11).
+2. Install the Intel platform services software.
+3. Follow the instructions at the [IAS website](https://api.portal.trustedservices.intel.com/EPID-attestation) to get
+   access to the IAS servers using a whitelisted SSL key.
+
+!!!Note
+   
+    1. You can *develop* enclaves on Linux, Windows, or macOS hosts. To use simulation mode in Windows and macOS, you 
+       need to install and run Docker. Intel requires that the CPU must at least support
+       [SSE4.1](https://en.wikipedia.org/wiki/SSE4#SSE4.1).
+    2. Conclave works only in mock mode on new Mac computers with Apple silicon due to a [known issue](known-issues.md).
+
+
+## Hardware support
+
+The machine needs support from both the CPU and the firmware. Please check with your hardware manufacturer to confirm 
+that your system supports SGX.
+
+You need to enable SGX in the BIOS/UEFI firmware screens for some machines. For others, a root user needs to activate 
+SGX. The [Enclave host API](api/-conclave%20-core/com.r3.conclave.host/-enclave-host/index.html) will try to 
+activate SGX if run with sufficient permissions.
+
+
+## Distribution support
+
+Intel supports the following Linux distros:
+
+* Ubuntu 18.04 LTS Desktop 64bits
+* Ubuntu 18.04 LTS Server 64bits
+* Ubuntu 20.04 LTS Desktop 64bits
+* Ubuntu 20.04 LTS Server 64bits
+* Red Hat Enterprise Linux Server release 7.6 64bits
+* Red Hat Enterprise Linux Server release 8.2 64bits
+* CentOS 8.2 64bits
+* Fedora 31 Server 64bits
+
+
+## Install the kernel driver and system software
+
+You can get the installers for the system software from Intel
+[here](https://01.org/intel-software-guard-extensions/downloads).
+
+Follow this
+[user guide](https://download.01.org/intel-sgx/sgx-linux/2.13.3/docs/Intel_SGX_Installation_Guide_Linux_2.13.3_Open_Source.pdf)
+to install the driver and the system software.
+
+Intel provides:
+
+* APT repositories for Ubuntu
+* Cross-distro installer binaries for other platforms, which set up the system software and install the kernel driver.
+
+!!!Important
+
+    You need to re-run the installer after upgrading the kernel.
+
+Alternatively, you can [compile the system software](https://github.com/intel/linux-sgx/releases/tag/sgx_2.13.3)
```

Comment:
THis is an old link, the new one should be this one
https://github.com/intel/linux-sgx/releases/tag/sgx_2.17.1